### PR TITLE
Fixed developer setup issue in open_registration_extension

### PR DESCRIPTION
**Pull request by JRosenkranz**
This PR fixes an issue where when running `python setup.py develop`, the `open_registration_extension` self contained example will not build due to the following:

```
error: 'synchronizeStream' overrides a member function but is not marked 'override'
```

### CODE REVIEW

**Review by albanD — APPROVED** [review]
Thanks!

**Comment by Skylion007** [comment]
Please sign the CLA

**Comment by JRosenkranz** [comment]
> Please sign the CLA

@Skylion007 Sorry for the wait. Currently waiting on an approval, should most likely have it ready Monday.

**Comment by JRosenkranz** [comment]
@Skylion007 CLA is done, sorry again for the wait.

**Review by spzala — APPROVED** [review]
lgtm Thanks @JRosenkranz

**Review by albanD — APPROVED** [review]
Thanks!

**Pull request closed without merging**
